frontend/src/app/main/ui/components/forms.cljs

```diff
@@ -74,7 +75,13 @@
                            "password"))))
 
         on-focus  #(reset! focus? true)
-        on-change (fm/on-input-change form input-name trim)
+        on-change (fn [event]
+                    (let [target (dom/get-target event)
+                          value  (if (or (= (.-type target) "checkbox")
+                                         (= (.-type target) "radio"))
+                                   (.-checked target)
```

Comment:
Maybe abstract it to the `app.util.dom` ns? With apropriate `^js` type hint.

Reply:
fixed